src/js/pre-logic.js

```diff
@@ -113,13 +182,13 @@ function crunchSprite(parentState) {
     if (isAnEdge) {
       switch(direction) {
       case 'left':
-        return {...state, x: parentState.board.rows};
+        return {...state, x: parentState.board.rows - 1};
```

Comment:
`rows` doesn't change. you could have `state.board.rows` and drop the `parentState` and the high order function